### ISSUE TITLE
.eslintrc.js is not loaded when using graphql plugin

### ISSUE THREAD

**Issue by TimSusa** [Microsoft/vscode-eslint#488]
Hey, 

I try to make use of the new graphql plugin for eslint. Therefore we are forced to use a eslintrc.js. I also tried to set eslint.validate with different settings for javascript and jsx. But:

On the console it keeps saying:

"Cannot read config file: /Users/blabla/.eslintrc.js Error: Unexpected token ..."

My config looks like this: 

```
const schema = require('./src/server/graphql/schema')
const { printSchema } = require('graphql')

const schemaString = printSchema(schema)

module.exports = {
  parser: 'babel-eslint',
  rules: {
    'graphql/template-strings': ['error', {
      env: 'apollo',
      schemaString
    }]
  },
  plugins: [
    'graphql'
  ]
}
```

**Comment by herr-vogel** [comment]
+1

**Comment by dbaeumer** [comment]
I assume that the graphql plugin uses JS syntax that needs a modern JS runtime. We load ESLint into VS Code node runtime which is still 7.9.0 (given by Electron). Does this reproduce for you if you execute this in a terminal using node 7.9.0?

If not can you provide me with a GitHub repository I can clone that demos this. 

P.S.: VS Code is working on moving to Electron 2.0 which will gives us a decent new JS runtime.

**Comment by herr-vogel** [comment]
Thanks for the clue, in schema we imported a file which used the spread operator. So it was the old JS runtime.

**Comment by dbaeumer** [comment]
Any objection to close the issue. This will be solved with the update to Electron 2.0.
